### ISSUE TITLE
Kibana-datasweet-formula support for kibana 7.0.1

### ISSUE THREAD

**Issue by dancristi4n** [datasweet/kibana-datasweet-formula#45]
Thank you so much for this plugin. The plugin is not compatible with the latest version of Kibana 7.0.1. Please support.

**Comment by lfroment-datasweet** [comment]
Hi, we will, soon. It is a tough job to follow up with every minor update....for free, so please be indulgent regarding delays.
Tweaking the Kibana version number in your package might do the trick. We did not test our latest version against 7.0.1 yet, but maybe it can spare you some waiting time.

**Comment by dancristi4n** [comment]
Hi. 
Forgive me if i sounded rude. It wasn't my intention. Thanks for reply and trick. I will try.

**Comment by lfroment-datasweet** [comment]
Hi, @dancristi4n. You did not sound rude, no problems! 
Fact is that the Elastic policy regarding plugins is clearly adding workload for small teams like we are. Add on top of it that breaking changes occur very often with every new minor or major release, AND the fact that Datasweet-Formula is a big hack of Kibana, then you get the perfect definition for hassle ! But we do it because we know that our plugin is useful and used (thousands of downloads per year). We wish more users like you take time to thank us and "star" this project.

**Comment by dancristi4n** [comment]
Hi.
I tried the trick but didn't works. 
I changed the Kibana version in my package file from "7.0.1" to "7.0.0". I downloaded the plugin and installed with successfull.
Initially, without return the version to "7.0.1", restarted the Kibana service and checked if dashboards, visualizations and anything else still working but some disappeared.
So, I returned the version to "7.0.1" and the dashboards and visualizations reappeared, but the function "datasweet formula" is disable.

Anyway, thanks.
 I already gave the "star" for this project. I'll wait for a new release.
